scripts/python/update_gencc.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+import subprocess 
+import argparse
+import pandas as pd
+import os
+import random 
+import sys
+import datetime
+import requests
+import json
+
+def get_ols(disease_name):
+    # only using mondo because mondo gives a close enough match with how we name diseases in G2P
+    statement = "No disease mim"
+    endpoint = 'http://www.ebi.ac.uk/ols/api/search?q='
+    ontology = '&ontology=mondo'
+    url = endpoint + disease_name + ontology
+    result = requests.get(url)
+    if result.status_code == 200:
+        final_result = json.loads(result.content)
+        response = final_result["response"]
+        documents = response["docs"]
+
+        mondo_id = [i["obo_id"] for i in documents if "obo_id" in i ]
+    else:
+        print("Connecting to the " + endpoint + " failed")
+        sys.exit()
+        pass
+        
+    if len(mondo_id) > 0:
+        return mondo_id[0]
+    else:
+        return statement
+
+allelic_requirement = {
+    "biallelic_autosomal" : "HP:0000007",
+    "monoallelic_autosomal" : "HP:0000006",
+    "monoallelic_X_hem"  : "HP:0001417",
+    "monallelic_Y_hem" : "HP:0001450",
+    "monoallelic_X_het" : "HP:0001417",
+    "mitochondrial" : "HP:0001427",
+    "monoallelic_PAR" : "HP:0000006",
+    "biallelic_PAR" : "HP:0000007"
+}
+con_category = {
+    "definitive" : "GENCC:100001",
+    "strong" : "GENCC:100002",
+    "moderate" : "GENCC:100003",
+    "limited" : "GENCC:100004"
+}
+
+subprocess.run(['bash', '../download_file.sh' ])
```

Comment:
It should have the full path to the script otherwise it only works for specific cases.